File: bigfuntool/src/main/java/com/yl/bigfuntool/IFBLoginListener.java
package com.yl.bigfuntool;

import android.support.annotation.Keep;

import com.facebook.FacebookException;

import org.json.JSONObject;

@Keep
public interface IFBLoginListener {
    /**
     * 取消
     */
    @Keep
    void onCancel();

    /**
     * 错误
     */
    @Keep
    void onError(FacebookException error);

    /**
     * 完成
     *
     * @param jsonObject
     */
    @Keep
    void onComplete(JSONObject jsonObject);
}

File: paytmlibrary/src/main/java/com/bigfun/tm/login/IFBLoginListener.kt
package com.bigfun.tm.login

import android.support.annotation.Keep
import com.facebook.FacebookException
import org.json.JSONObject

@Keep
interface IFBLoginListener {
    /**
     * 取消
     */
    @Keep
    fun onCancel()

    /**
     * 错误
     */
    @Keep
    fun onError(error: FacebookException)

    /**
     * 完成
     * @param jsonObject
     */
    @Keep
    fun onComplete(jsonObject: JSONObject?)
}